src/web/ScrollControls.tsx

```diff
@@ -108,7 +111,7 @@ export function ScrollControls({
     el.style[horizontal ? 'overflowX' : 'overflowY'] = 'auto'
     el.style[horizontal ? 'overflowY' : 'overflowX'] = 'hidden'
     el.style.top = '0px'
-    el.style.left = '0px'
+    el.style.left = hideScrollbar ? '15px' : '0px'
```

Comment:
shouldn't it be reversed?

Reply:
really sorry but been busy with some unexpected work. Going to get back to you as soon as I can with an example, but may be another couple of weekends